.github/workflows/ci.yaml

```diff
@@ -3,7 +3,7 @@ on:
   - push
   - pull_request
 env:
-  golang-version: '1.13'
+  golang-version: '1.14'
```

Comment:
Why not migrate to go `1.15`?

Reply:
it's possible to bump `1.15`. However, the issue https://github.com/prometheus-operator/kube-prometheus/issues/493 asked 1.14. Should I bump `1.15`?

Reply:
I think it's safe to bump directly to `1.15`. It's already a stable release. However, this is not a decision for me to make 😅 Let's give a chance to maintainers to review.

Reply:
Let's bump to 1.15 ;)